src/test/java/codeu/controller/AdminServletTest.java

```diff
@@ -54,9 +53,9 @@ public void setup() {
     Mockito.when(mockRequest.getSession()).thenReturn(mockSession);
 
     mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
-     /* When AdminServlet tries to forward to the dispatcher of "/WEB-INF/view/admin.jsp", just
-      * return the mock dispatcher.
-      */
+    /* When AdminServlet tries to forward to the dispatcher of "/WEB-INF/view/admin.jsp", just
```

Comment:
Are comments supposed to be
```java
/* like
 * this?
 */
```
or
```java
/**
 * like
 * this?
 */
```

Reply:
Not sure it matters for inline comments. The latter is for javadocs I believe